sources/packages/backend/apps/workers/src/controllers/application/application.controller.ts

```diff
@@ -168,4 +186,78 @@ export class ApplicationController {
       });
     }
   }
+
+  /**
+   * Processes the application change request approval.
+   * Set the application edit status to 'Change pending approval' to allow waiting
+   * till the Ministry approves or declines the changes.
+   * @returns most updated edit status.
+   */
+  @ZeebeWorker(Workers.ApplicationChangeRequestApproval, {
+    fetchVariable: [APPLICATION_ID],
+    maxJobsToActivate: MaxJobsToActivate.Normal,
+  })
+  async applicationChangeRequestApproval(
```

Comment:
I see that this worker has been used for 2 different purposes, 1. Update the Edit Status 2. Return the current edit_status. 

Even though, if there is additional effort to introduce another worker, I feel using a worker to `Update Edit Status` and `Verify Approval` sounds better to me. Open for discussion.

Reply:
The idea was to follow a similar approach from `verify-application-exceptions`, which is also for "two purposes" and not to mention some other workers like `verify-assessment-calculation-order` and `workflow-wrap-up`.
In terms of a worker executing a single responsibility, yes, looks better, but we usually take into consideration the workflow cost of executing a worker call/process. Does it make sense?
I would be more inclined to create an additional worker if one of those suggested separations would be reused in more than one place.

Reply:
I agree that unless we have more purpose for the new worker we can follow the current approach in PR.